app/models/user_preference.rb

```diff
@@ -0,0 +1,15 @@
+class UserPreference < ApplicationRecord
```

Comment:
I've created a `user preferences` model according to [this suggestion](https://community.theforeman.org/t/user-preferences/12007/12)
I think it would be better to do `table preferences` migration into this in a follow up PR.